build-tools/packages/build-tools/package.json

```diff
@@ -30,6 +30,7 @@
 		"eslint:fix": "eslint --format stylish src --fix --fix-type problem,suggestion,layout",
 		"format": "npm run format:biome",
 		"format:biome": "biome check --write .",
+		"generate:biomeConfigTypes": "json2ts --input node_modules/@biomejs/biome/configuration_schema.json --output src/common/biomeConfigTypes.d.ts",
```

Comment:
Can be run manually when needed to update the types from the schema.

Reply:
Unfortunately `json-schema-to-typescript` doesn't work. The only `bin` the package defines is json2ts. I could `pnpm
patch` the package but that seems overkill. I'll rename this task to include the `json-schema-to-typescript` string.
That will at least help when searching for the string.

Reply:
It's okay. Probably won't come up often and search for the file name would find this too.